### ISSUE TITLE
print(CompositeCost) fails / not expected behaviour

### ISSUE THREAD

**Issue by retowildhaber** [lmlib/lmlib#8]
```
ccost = lm.TSLM.create_cost( ab=(-15, 30), gs=(50, 50) )
print(ccost)
```

produces output

`<lmlib.statespace.cost.CompositeCost object at 0x126448790>`

instead printing (dump_tree()) of internal model structure as proposed in documentation of CompositeCost, see here:

.../doc/_build/html/api/statespace/_autosummary/lmlib.statespace.cost.CompositeCost.html?highlight=compositecost#lmlib.statespace.cost.CompositeCost

Tasks:
1. either print(cc) or print(cc.dump_tree) should print the model strucutre. Which is better?
2. please add correct print output to example in TSLM() documentation. See in branch dev:

.../doc/_build/html/api/_autosummary/lmlib.statespace.applications.TSLM.html#lmlib.statespace.applications.TSLM

**Comment by fwaldmann** [comment]
Bugfix, Branch: dev 

Rev. 10e5c8fd818ecc04987ea8e5164848030389b330

**Comment by fwaldmann** [comment]
Fixed in release v2.1.0
rev. 841cf96ec478daacf2fdbf03f14607f5151ec5ec
